leaderboards/leaderboard_challenges.tex

```diff
@@ -0,0 +1,160 @@
+
+Participating teams must start the robot behaviour of their competing robot via a single chest button
+press (this mimics the chest button behaviour of switching between playing and penalized state).
+The participating robot must navigate autonomously and cannot be remote controlled during the challenge
+
+\subsubsection{Scoring}
+Time starts from when the chest button is pressed, to when to robot entirely crosses the target touchline.
+
+Teams will take 3 attempts at navigating both runs, with the sum of the lowest time for each
+run being recorded in the leaderboard. Times will be rounded to the nearest second.
+
+\subsection{Best Kick Leaderboard}
+This leaderboard will measure how far and how accurate a NAO will be able to kick a standard
+SPL soccer ball.
+
+\subsubsection{Setup}
+This leaderboard challenge will take place on a standard SPL field, and requires 1 active robot and 1 ball.
+The ball will be placed on the centre of the edge of the goal area, in line with the penalty mark.
+The competing robot will be placed in the centre of the goaline facing the ball
+\begin{figure}[t]
+    \centerline{\includegraphics[width=\columnwidth]{figs/kick_leaderboard.pdf}}
+    \caption{Best Kick Leaderboard: Robot completing the attempt is shown in blue, targets are shown in green, ball placement is shown in red}
+    \label{fig:kick_leaderboard}
+\end{figure}
+\subsubsection{Challenge Execution}
+Teams will be given 3 attempts at kicking to each of the 3 target postitions (illustrated in \cref{fig:kick_leaderboard}):
+\begin{itemize}
+    \item The centre mark in the centre circle
+    \item The penalty mark on the opposite side of the field
+    \item A corner on the opposite side of the field
+\end{itemize}
+In each attempt, participating teams must start the robot behaviour of their competing robot via a single chest button
+press. The robot then has \qty{\PenaltyShootoutKickTime}{\second} to kick the ball towards the current target.
+The competing robot must not touch the ball a second time after the ball has clearly moved,
+otherwise the attempt ends immediately without scoring.
+Once the ball has come to a complete stop, the distance from the ball to the target is measured.
+
+Code changes are only allowed when changing targets.
+\subsubsection{Scoring}
+The sum of the smallest distance from each target will be recorded in the leaderboard.
+Distances are rounded to the closest centimetre.
+
+\subsection{Most Passes Leaderboard}
+This leaderboard measures the highest number of successful passes a team can
+complete in a given timeframe, emphasizing passing accuracy, speed, and coordination.
+
+\subsubsection{Setup}
+This leaderboard challenge will take place on a standard SPL field and requires 2 competing robots
+and 1 standard SPL soccer ball.
+
+Competing robots will be placed on the touchlines on opposite sides of the same half of the field,
+in line with the penalty mark. The ball will be placed on the corner of the penalty area.
```

Comment:
It may be good to clarify that it could be placed on "either corner" of the penalty area, rather than "the corner"?